.github/workflows/build.yml

```diff
@@ -84,11 +86,6 @@ jobs:
       - name: coursier-cache-action
         uses: coursier/cache-action@v5
 
-      - name: Extract branch name
-        shell: bash
-        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
```

Comment:
IIRC you were wondering about this, I didn't really find an explanation but I did see this exact code here FWIW https://github.com/orgs/community/discussions/35994#discussioncomment-3896937